### PR TITLE
JW-345 Fixed the order of uninitializing the switch

### PR DESCRIPTION
**Pull request by Attil**

### CODE REVIEW

File: windows/vrouter_mod.c [hunk 1]
```diff
@@ -180,17 +180,17 @@ SxExtUninitialize(PDRIVER_OBJECT DriverObject)
 void
 SxExtUninitializeVRouter(struct vr_switch_context* ctx)
 {
-    if (ctx->device_up)
```
Comment (mmasztalerczuk) [inline windows/vrouter_mod.c]: You are not cleaning ctx->device_up and others variables

**Comment by mmasztalerczuk** [comment]
Line 251 ctx = ExAllocatePoolWithTag(NonPagedPoolNx, sizeof(struct vr_switch_context), SxExtAllocationTag) can create blue screen if something go wrong in for example 265

**Comment by Attil** [comment]
Janie, przetestuj to proszę

**Comment by m-kostrzewa** [comment]
Janie, przetestuj to proszę

**Pull request merged**